content/about/notes/1.1/index.md

```diff
@@ -49,23 +49,34 @@ Istio 1.1.
 
 ## Traffic management
 
-- **New `Sidecar` Resource**. Added support to limit the set of services
-  visible to sidecar proxies in a given namespace using the `Sidecar` resource.
-  This limit reduces the amount of configuration computed and transmitted to
-  the proxy. On large clusters, we recommend adding a sidecar resource per
-  namespace.
-
-- **Restrict Visibility of Networking Resources**. Added the new `exportTo`
-  field to all networking resources which lets you control the visibility of
-  individual resources to specific namespaces.
+- **New [`Sidecar`](/docs/concepts/traffic-management/#sidecars) Resource**. Enables more fine-grained
+  control over the behavior of the sidecar proxies attached to workloads within a namespace. In 
+  particular adds support to limit the set of services a sidecar will send traffic to. This  
+  reduces the amount of configuration computed and transmitted to
+  the proxy, improving startup time, resource consumption and control-plane scalability. 
+  For large deployments, we recommend adding a sidecar resource per namespace.
+
+- **Restrict Visibility of Services**. Added the new `exportTo` feature which allows
+  service owners to control which namespaces can reference their services. This feature is 
+  added to `ServiceEntry`, `VirtualService` and is also supported on a Kubernetes Service via the
+  `networking.istio.io/exportTo` annotation. 
+  
+- **Namespace-Scoped Service References**. When referring to a service (host) we use DNS-style name matching
```

Comment:
per offline discussion, I wouldn't portray this concept as an alt way of referring to services just yet, until we resolve all the issues associated with it. For now, I would simply say that gateways can refer to specific virtual services in a namespace using the namespace/host syntax.. Similarly, virtual services can refer to specific gateways in a namespace using the namespace/gatewayName syntax..

The sidecar's format is not a change over existing. It is completely new and syntax/semantics are conveyed in the docs..

Reply:
Done. PTAL